pyproject.toml

```diff
@@ -28,7 +28,7 @@ dagster-meltano = ">=1.0.0"
 # Dependencies from Meltano EDK, remove when dependency fixed
 structlog = "^21.2.0"
 PyYAML = "^6.0.0"
-pydantic = "^1.9.0"
+pydantic = "<2.10,>=2"

```

Comment:
@joshua-janicas you'll probably want to run `poetry lock`

Reply:
Thank you - and done!